images/releng-ci-bazel/cloudbuild.yaml

```diff
@@ -1,10 +1,30 @@
 ---
 # See https://cloud.google.com/cloud-build/docs/build-config
 timeout: 1200s
+
+#### SECURITY NOTICE ####
+# Google Cloud Build (GCB) supports the usage of secrets for build requests.
+# Secrets appear within GCB configs as base64-encoded strings.
+# These secrets are GCP Cloud KMS-encrypted and cannot be decrypted by any
+# human or system outside of GCP Cloud KMS for the GCP project this encrypted
+# resource was created for.  Seeing the base64-encoded encrypted blob here is
+# not a security event for the project.
+# More details on using encrypted resources on Google Cloud Build can be found
+# here:
+# https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials
+#
+# (Please do not remove this security notice.)
+secrets:
+  - kmsKeyName: projects/kubernetes-release-test/locations/global/keyRings/anago/cryptoKeys/k8s-release-robot
+    secretEnv:
+      GITHUB_TOKEN: CiQAveh8wGJqpEkcVluO3tBntlehynOxiOPDD9u1XCONx3vuozoSUgCMoh/IJuNkqhcDTP2om2tyOStft8myMSrvnGd7NvTo+H+fI0EkLMNzkVOHxl/C0piktBm74uN70QVE+e4TTa9wwA//qpiSm/UuqYmEYeMIpyY=
```

Comment:
This GitHub token is from a robot? Not your personal correct?

Reply:
Yes, it's the same as used in the GCB builder image:
https://github.com/kubernetes/release/blob/d20c2b40d46c5fdc0d1a2f14cee647841c86e165/gcb/stage/cloudbuild.yaml#L17

Reply:
cool, thanks for the clarification